iscript/src/iscript/mac.py

```diff
@@ -515,17 +516,22 @@ async def create_one_notarization_zipfile(work_dir, all_paths, path_attrs=("app_
         app.check_required_attrs(required_attrs)
         for path_attr in path_attrs:
             app_paths.append(os.path.relpath(getattr(app, path_attr), work_dir))
-    await run_command(["zip", "-r", zip_path, *app_paths], cwd=work_dir, exception=IScriptError)
+    if sign_config["zipfile_cmd"] == "zip":
+        await run_command(["zip", "-r", zip_path, *app_paths], cwd=work_dir, exception=IScriptError)
+    elif sign_config["zipfile_cmd"] == "ditto":
+        await run_command(["ditto", "-c", "-k", "--sequesterRsrc", "--keepParent", "0", zip_path], cwd=work_dir, exception=IScriptError)
```

Comment:
If we're going to move commands out to the config, what do you think about putting all the args in the config too?

I also wonder if we should just try using `ditto` for Firefox and avoid this branch altogether.

Reply:
> If we're going to move commands out to the config, what do you think about putting all the args in the config too?

Hm, could do.

> I also wonder if we should just try using `ditto` for Firefox and avoid this branch altogether.

The reason I branched was Nick and I tried using `ditto` for Firefox and ended up with a ton of extra cruft in the .app that ended up breaking notarization. I don't know if we used these exact args, though, so it might be worth trying again.

Reply:
Maybe we'll give it a try once and see if it works? If it breaks notarization that's a pretty clear and obvious failure.

Reply:
I was able to notarize Firefox successfully! It passed `codesign -vvvv` and `spctl --assess --verbose` checks. We need to test MozillaVPN signing, and then we're ready for review/landing/rollout.

Reply:
I tested this for MozillaVPN this morning by:
* Replacing the `iscript` package in the scriptworker mac-v3-signing1 virtualenv with the version from your branch
* Tweaking `task.json` to set `product` to `mozillavpn`
* Signing a mozilla vpn dmg.

AFAICT it worked fine. I'm good with landing this at your convenience.